oem/ibm/requester/dbus_to_file_handler.cpp

```diff
@@ -44,8 +44,16 @@ void DbusToFileHandler::sendNewFileAvailableCmd(uint64_t fileSize)
                                     PLDM_NEW_FILE_REQ_BYTES);
     auto request = reinterpret_cast<pldm_msg*>(requestMsg.data());
     // Need to revisit this logic at the time of multiple resource dump support
-    uint32_t fileHandle = 1;
 
+    std::string pathString = static_cast<std::string>(resDumpCurrentObjPath);
+    std::string filename = fs::path(pathString).filename().string();
+
+    char* end;
+    uint32_t fileHandle = std::strtoul(filename.c_str(), &end, 16);
+
+    info(
+        "File handle in sendNewFileAvailableCmd that we encode new_file_req is {FILEHANDLE}",
```

Comment:
Pls change FILEHANDLE to match with filename

Reply:
done